src/components/workspace/side-content/AutograderCard.tsx

```diff
@@ -47,45 +24,40 @@ class AutograderCard extends React.Component<AutograderCardProps, {}> {
       }
     };
 
+    const isCorrect =
+      this.props.testcase.actual !== undefined
+        ? // tslint:disable-next-line
```

Comment:
why is tslint disabled here?

Reply:
Refactored the code. I had an issue with === equality returning false.